support/cas-server-support-pm-webflow/src/test/java/org/apereo/cas/pm/web/flow/actions/ValidatePasswordResetTokenActionTests.java

```diff
@@ -31,6 +39,24 @@ void verifyInvalidTicket() throws Throwable {
         assertEquals(CasWebflowConstants.TRANSITION_ID_INVALID_PASSWORD_RESET_TOKEN, validatePasswordResetTokenAction.execute(context).getId());
     }
 
+    @Test
+    void verifyGoodTicket() throws Throwable {
+        ClientInfoHolder.setClientInfo(new ClientInfo());
+        val id = UUID.randomUUID().toString();
+        val ticket = new TransientSessionTicketImpl();
+        ticket.setId(id);
+        ticket.setCreationTime(ZonedDateTime.now(ZoneId.systemDefault()));
+        ticket.setExpirationPolicy(NeverExpiresExpirationPolicy.INSTANCE);
+        val query = PasswordManagementQuery.builder()
+                .username("casuser").email("casuser@email.com").build();
+        val token = passwordManagementService.createToken(query);
+        ticket.getProperties().put("token", token);
+        ticketRegistry.addTicket(ticket);
+        val context = MockRequestContext.create();
```

Comment:
If possible, always best to use `val context = MockRequestContext.create(applicationContext);`.

PS If other tests miss this in this class, please update them as well.

Reply:
Will do.